src/script/cpatch

```diff
@@ -195,11 +193,13 @@ if [ $rgw -eq 1 ] || [ $all -eq 1 ]; then
     [ $strip -eq 1 ] && strip $TMP/rgw/*
     dockerfile+=$'ADD rgw /usr/bin\n'
 
-    RGWLIBS="libradosgw.so*"
+    RGWLIBS="librados.so.* libceph-common.so.*"
     mkdir -p $TMP/rgwlib
-    for f in lib/$RGWLIBS; do cp $f $TMP/rgwlib ; done
+    for f in $RGWLIBS; do cp lib/$f $TMP/rgwlib ; done
     [ $strip -eq 1 ] && strip $TMP/rgwlib/*
     dockerfile+=$'ADD rgwlib /usr/lib64\n'
```

Comment:
I realised that this was duplicating work if the --all parameter was specified, but struggled to find a neat way to avoid it.  I like the idea of copying all binaries and libraries to the same place and then moving them at the end so I'll make that change,